app/(Home)/Section5.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+
+import Link from 'next/link'; 
+import { SectionWrapper } from '@/components/HOC'
+import FormSection from '@/components/Idea_Section/FormSection';
+
+function Section_5() {
+  return(
+    <>
+    <h1 className="px-6 mt-32  mb-32 font-normal text-3xl sm:text-6xl tracking-wide text-center">
+          // HAVE AN IDEA ?
+    </h1>
+    <section className="flex flex-col sm:flex-col md:flex-col lg:flex-row justify-around mb-32" id="Section_8">
+      <div className="w-full lg:w-1/2">
+      <section className="p-6 w-full md:p-24" id="#Section_1">

```

Comment:
`<section className="p-6 w-full md:p-24" id="#Section_1">` this isn't section 1, fix.